templates/iam_policy/draft_metadata_validator_lambda.json.tpl

```diff
@@ -38,6 +38,14 @@
         "kms:Decrypt"
       ],
       "Resource": "${kms_key_arn}"
+    },
+    {
+        "Effect": "Allow",
+        "Action": [
+         "ecr:BatchGetImage",
+         "ecr:GetDownloadUrlForLayer"
+     ],
+     "Resource": "arn:aws:ecr:eu-west-2:${management_account_id}:repository/draft-metadata-validator"
```

Comment:
See comment above about changing the account id variable name to `ecr_account_id` for naming consistency

Reply:
done